apps/core/scenes/bookingDetail/sectorDetails/segments/SegmentMap.js

```diff
@@ -62,15 +129,15 @@ class SegmentMap extends React.Component<Props> {
         }}
         zoomEnabled={false}
```

Comment:
Btw are you sure that we shouldn't allow users to zoom in? 

It's hard to say if the point is in Vienna or Bratislava:
<img width="340" alt="screenshot 2019-02-26 at 13 10 18" src="https://user-images.githubusercontent.com/858321/53411849-eb45b500-39c7-11e9-97c4-0e32169e2ee4.png">

Reply:
I'm ok with disabled zoom. Window is quite small for any fiddling with map.
We can later implement expanded movable/scrollable map on tap like in mobile app.

Reply:
Current usage in (native)app is the map not zoomable or dragable. When you click on it, you open map in full-screen modal.

Reply:
> When you click on it, you open map in full-screen modal.

I'm OK with that, it makes sense this way